test/cases/regression-old-worker-new-composer.sh

```diff
@@ -13,9 +13,9 @@ ARTIFACTS="${ARTIFACTS:-/tmp/artifacts}"
 source /usr/libexec/osbuild-composer-test/set-env-variables.sh
 source /usr/libexec/tests/osbuild-composer/shared_lib.sh
 
-# Only run this on x86 and rhel8 GA
+# Only run this on x86
```

Comment:
That RHEL GA part was correct, we still check for it in the condition, just the version was supposed to be both 8 and 9.

Reply:
I think it could be clear from the error mesage